proxy/http/HttpTransact.cc

```diff
@@ -757,15 +757,15 @@ how_to_open_connection(HttpTransact::State *s)
     break;
   }
 
-  s->cdn_saved_next_action = HttpTransact::SM_ACTION_ORIGIN_SERVER_OPEN;
+  HttpTransact::StateMachineAction_t connect_next_action = HttpTransact::SM_ACTION_ORIGIN_SERVER_OPEN;
 
   // Setting up a direct CONNECT tunnel enters OriginServerRawOpen. We always do that if we
   // are not forwarding CONNECT and are not going to a parent proxy.
   if (s->method == HTTP_WKSIDX_CONNECT) {
     if (s->txn_conf->forward_connect_method == 1 || s->parent_result.result == PARENT_SPECIFIED) {
-      s->cdn_saved_next_action = HttpTransact::SM_ACTION_ORIGIN_SERVER_OPEN;
+      connect_next_action = HttpTransact::SM_ACTION_ORIGIN_SERVER_OPEN;
```

Comment:
Isn't it already that value? Shouldn't this check only for having to change to `SM_ACTION_ORIGIN_SERVER_RAW_OPEN`?

Reply:
Yes, I was trying to minimize code change to simplify verification that the code change was not a functional change.  But you are correct, this could be further simplified.